File: java/class 5/Question2.java
import java.util.Scanner;

public class Question2 {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);
    System.out.print("Enter your age: ");
    int age = input.nextInt();
    System.out.println(age >= 18? "Access granted": "You are too young to go beyond this point" );
    input.close();
  }
}

File: python/class 5/question 2.py
age=int(input("Please enter your age: "))
if age <= 18:
  print('You are too young to go beyond this point.')
elif age>18:
  print('You have been granted access.')






#alternatively you could do 
print('You are too young to go beyond this point.' if (age:=int(input("Please enter your age: "))) <= 18 else "You have been granted access.")